BASE_IMAGE

```diff
@@ -1 +1 @@
-docker.io/kindest/base:v20230310-474355fc
+quay.io/kviswana/base:v20240611-d897688d
```

Comment:
To be moved to quay.io/powercloud.

Reply:
where is the code to build/push this image? lets first fix that as a prerequisite and then work on this PR